docs/content/docker.md

```diff
@@ -536,6 +536,13 @@ sudo curl -H Content-Type:application/json -XPOST -d {} --unix-socket /run/docke
 ```
 though this is rarely needed.
 
+If the plugin fail to work properly, and only In last resort after you tried diagnosing with the above methods, you can try clearing the state of the plugin. **Note that all exiting rclone docker volume will probably have to be recreated** (This might be needed as a reinstall don't cleanup existing state files to allow for easy restoration, as stated above.)
```

Comment:
`only In last resort` : Change to lowercase `i` - but even better, I think, it is to write `as a last resort`.
`Note that all exiting rclone docker volume` : `exiting` should be `existing`, I assume? And make plural `volumes`.
`to be recreated**`: Add ending punctuation, `to be recreated.**`